src/Bicep.Core.Samples/Files/baselines/Functions_LF/main.formatted.bicep

```diff
@@ -31,15 +36,10 @@ func test3() object => loadYamlContent('./repro-data.json')
 func test4() string => loadFileAsBase64('./repro-data.json')
 
 // validate formatter works (https://github.com/Azure/bicep/issues/12913)
-func a(
-  ____________________________________________________________________________________________ string
-) string => 'a'
-func b(
-  longParameterName1 string,
-  longParameterName2 string,
-  longParameterName3 string,
-  longParameterName4 string
-) string => 'b'
+func a(____________________________________________________________________________________________ string) string =>
+  'a'
+func b(longParameterName1 string, longParameterName2 string, longParameterName3 string, longParameterName4 string) string =>
```

Comment:
This change is due to the increase of width to 120?

Reply:
Yes.